### PR TITLE
Update next-sass/README.md for next@7+

### PR DESCRIPTION
**Pull request by Jungwoo-An**
#290

### CODE REVIEW

File: packages/next-sass/readme.md [hunk 1]
```diff
@@ -133,25 +133,26 @@ module.exports = withSass({
...
-        </Head>
-        <body>
-          <Main />
-          <NextScript />
-        </body>
-      </html>
-    )
-  }
-}
+If you are using version less than next@7.0.0
```
Comment (timneutkens) [inline packages/next-sass/readme.md]: I think it's best to just remove this from the docs.

Comment (Jungwoo-An) [reply packages/next-sass/readme.md]: You say is If you are using version less than next@7.0.0?

Comment (timneutkens) [reply packages/next-sass/readme.md]: What I mean is that this whole section can be removed (adding the _document.js), as the docs are always for the latest version.

Comment (Jungwoo-An) [reply packages/next-sass/readme.md]: Ok. i'll update this pr

**Comment by Jungwoo-An** [comment]
Thanks @timneutkens. I removed _document.js relation content of all style preprocessor

**Pull request merged**